tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -72,10 +74,12 @@ tests:
       - stm32h573i_dk
     integration_platforms:
       - nucleo_g474re
-  drivers.spi.stm32_spi_dma_no_nocache.loopback:
+  drivers.spi.stm32_spi_dma_dt_nocache_mem.loopback:
+    # this test case is for when nocache memory region is defined in DT
+    # using `zephyr,memory-attr = < DT_MEM_ARM_MPU_RAM_NOCACHE)>`
     extra_args:
-      - OVERLAY_CONFIG="overlay-stm32-spi-dma-no-nocache.conf"
-    filter: CONFIG_SOC_FAMILY_STM32
+      - OVERLAY_CONFIG="overlay-stm32-spi-dma-dt-nocache-mem.conf"
+    filter: CONFIG_SOC_SERIES_STM32H7X
```

Comment:
No need to test on F7 ?

Reply:
I couldn't get it to run on an F7 board, since the ones I have only have 1 SRAM region, & designating only a chunk of that RAM as non-cacheable does not work for SPI (but surprisingly works for UART). Nevertheless, I don't think the test case should be restricted to H7.